### DISCUSSION TITLE
Improve the Relative dropdown functionality for Uses in Traits and Talents (see comments)

### DISCUSSION THREAD

**Discussion by Savantford** [Savantford/foundry-weirdwizard#293] [Polishing and Improvements]
> Burning River — 25/06/2024 14:11
Could you also change this to be equal to your path?  ie, novice 1, expert 2,master 3
a lot of magic talents work like that
![image](https://github.com/Savantford/foundry-weirdwizard/assets/36486145/9fd355c0-ecfc-49b8-a31b-6a27fbaf0e62)
Feather — 25/06/2024 21:02
Can you send an example of wording involving that? I havent read over all of the magic talents yet
Burning River — 26/06/2024 02:40
"You can use this talent once, twice at level 3, and three times at level 7. You regain expended uses after you rest."
I think there is around 1+1/3 per tradition on average

**Comment by PETHTH** [comment]
@Savantford , just played around with implementing the 3rd and 7th level and got it working fine.
Are you looking for some light contributions or do you prefer to keep things within the core development team?

**Reply by Savantford** [reply]
sorry for the very late response!

I was so focused on 6.0.0 release that I had to turn on tunnel vision in order to get it out ASAP!

I saw your pull request and will review it probably this week. I'll try to ship it 

every contribution is welcome! especially like those that attempts to clear the huge backlog!

we do discuss a lot of it in our Discord server, in the #development-chat channel: https://discord.gg/X5XustKpe4

please join us there so we can have better comms than here!

and thank you very much for the contribution!!!

**Comment by Savantford** [comment]
Implemented in 6.1.0 by @PETHTH!
